packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-form.tsx

```diff
@@ -120,23 +120,6 @@ export const AdditionalKycInfoForm = observer(({ setError }: TAdditionalKycInfoF
                                     {...fields.tax_identification_number}
                                     data_testId='dt_tax_identification_number'
                                 />
```

Comment:
cant we remove import of  OECD_TIN_FORMAT_URL

Reply:
Removed